doc/getting_started/getting_started.rst

```diff
@@ -339,9 +266,38 @@ and then:
 You can run ``zephyr/zephyr.exe --help`` to get a list of available
 options.  See the :ref:`native_posix` document for more information.
 
-This executable can be instrumented like any other Linux process. For ex. with gdb
-or valgrind.
-Note that the native port is currently only tested in Linux.
+This executable can be instrumented using standard tools, such as gdb or
+valgrind.
+
+.. rubric:: Footnotes
+
+.. [#homebrew_user]
+
+   For details, see
+   https://docs.brew.sh/Homebrew-and-Python#note-on-pip-install---user.
+
+.. [#tools_native_posix]
+
+   Usually, the toolchain is a cross-compiler and related tools which are
+   different than the host compilers and other programs available for
+   developing software to run natively on your operating system.
+
+   One exception is when building Zephyr as a host binary to run on a POSIX
+   operating system. In this case, you still need to set up a toolchain, but it
+   will provide host compilers instead of cross compilers. For details on this
+   option, see :ref:`native_posix`.
+
+.. [#board_misnomer]
+
+   This has become something of a misnomer over time. While the target can be
+   -- and often is -- a microprocessor running on its own dedicated hardware
+   board, Zephyr also supports using QEMU to run targets built for other
+   architectures in emulation, targets which produce native host system
```

Comment:
```suggestion
   architectures in emulation.  These build targets produce native host system
```

Reply:
That's not quite right. The 'targets which produce native host system binaries' refers specifically to the `native_posix` board, which is a separate configuration from the QEMU ones. (The QEMU targets do not produce native host system binaries in general; they produce binaries in the target instruction set which are dynamically translated to the host processor instruction set at runtime by QEMU).

Reply:
OK.  I was trying to break the v-e-r-y long sentence into two.